packages/ui-badge/src/Badge/index.tsx

```diff
@@ -133,11 +133,11 @@ class Badge extends Component<Props> {
     formatOverflowText: (_count: number, countUntil: number) =>
       `${countUntil - 1} +`
   }
-  _defaultId: string
```

Comment:
was this removed by accident?

Reply:
You are indeed correct, it was deleted by mistake